crates/lgn-graphics-renderer/src/components/render_surface.rs

```diff
@@ -669,3 +676,136 @@ pub(crate) fn reflect_viewports(
         }
     }
 }
+
+#[allow(clippy::needless_pass_by_value)]
+pub fn build_display_lists(
```

Comment:
This is not the right place for this function (system), but I don't know where to put it. In renderer/lib.rs? Open to suggestions.